config/initializers/stripe.rb

```diff
@@ -1 +1,17 @@
 Stripe.api_key = ENV["STRIPE_SECRET_KEY"]
+
+StripeEvent.configure do |events|
+  events.subscribe 'charge.succeeded' do |event|
+    user = User.where(customer_token: event.data.object.customer).first
+    Payment.create(
+      user: user,
+      amount: event.data.object.amount,
+      reference_id: event.data.object.id
+    )
+  end
+
+  events.subscribe 'charge.failed' do |event|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.